sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample_GetAndListOperations.md

```diff
@@ -0,0 +1,75 @@
+# Get and List Document Model Operations
+
+This sample demonstrates how to Get and List all document model operations (succeeded, in-progress, failed) associated with the Form Recognizer resource. Note that operation information only persists for 24 hours.
+If the operation was successful, the document model can be accessed using `GetModel()` or `ListModels()` APIs.
```

Comment:
Do you want to mention about the error scenario, i.e if failed operation you can access the errors via result.error?

Reply:
Good idea! Will add

Reply:
added